dune-project

```diff
@@ -125,7 +127,7 @@ This package provides the Coq Reference Manual."))
  (name coq)
```

Comment:
Since this definition does not correspond to the Opam package (which explains why `dune build -p coq` succeeds, despite not installing `coq-stdlib`), it would be better to give this Dune package a different name, to avoid any confusion.

Reply:
I don't know if it's even possible to give dune package a name that differs from the generated opam package.

Reply:
I don't understand what we are trying to achieve by preserving the Coq package in the Coq repo.

Reply:
> to give dune package a name that differs from the generated opam package.

My point is that it is no longer the generated opam package, so there is no reason for them to have the same name.

> what we are trying to achieve by preserving the Coq package in the Coq repo.

As far as I am concerned, as a developer, it means that I can do `opam pin add . --with-version=8.21.0` to get an environment in which I can then install user libraries. If the `coq.opam` file is removed, I am not sure what command line I am supposed to use instead.

Reply:
Ok, then let's keep things as they are. I tried removing the dune package while keeping the opam file but that makes dune scream.